tests/drivers/clock_control/nrf2_clock_control/boards/nrf54h20dk_nrf54h20_cpuapp.overlay

```diff
@@ -0,0 +1,23 @@
+&hfxo {
+	status = "okay";
+};
+
+&lfxo {
+	status = "okay";
+};
+
+&fll16m {
+	status = "okay";
+};
+
+&hsfll120 {
+	status = "okay";
+};
+
+&cpuapp_hsfll {
+	status = "okay";
+};
+
+&lfclk {
+	status = "okay";
+};
```

Comment:
These are not required as they are all enabled by default in the boards dts already :)